scss/_variables.scss

```
@@ -1047,10 +1047,11 @@ $btn-social-networks: (
 
 // scss-docs-start form-text-variables
 $form-text-margin-top:                  .375rem !default; // Boosted mod
-$form-text-font-size:                   null !default;
+$form-text-font-size:                   $small-font-size !default;
 $form-text-font-style:                  null !default;
-$form-text-font-weight:                 null !default;
-$form-text-color:                       null !default;
+$form-text-font-weight:                 $font-weight-bold !default; // Boosted mod
+$form-text-line-height:                 $line-height-sm !default; // Boosted mod
+$form-text-color:                       $text-muted !default;
```

This PR has been opened long time ago. Now that we have the color modes, `$text-muted` is deprecated, so let's use `var(--#{$prefix}secondary-color)` instead:

```suggestion
$form-text-color:                       var(--#{$prefix}secondary-color) !default;
```